Launcher/config.json

```diff
@@ -274,6 +278,20 @@
       "history-provider": "BrokerageHistoryProvider"
     },
 
+    "live-binance": {
```

Comment:
AWESOME! thanks for adding the `live-binance` config environment

Reply:
You are welcome! 😄 
P.S. could you resolve this conversation if you don't expect any changes here